.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,65 @@
+<!--
+Please fill in a a meaningful and concise issue title in the title field above.
+It helps us to classify the issue even  before reading the body and handling the
+issue a bit better.  Prefix the title with labels you find approciate to
+describe the issue like "[Bug]" in case of a bug, "[Feature]" for a feature
+request.
+
+Examples:
+  * Horrible: "HELP!!!"
+  * Not really okay: "Issue with fleets"
+  * Good: "[Bug] Unable to click fleet"
+  * Great: "[Bug] Unable to select fleet in galaxy map"
+-->
+
+
+### Environment
+<!--
+You don't need to provide the information in section when they are not useful in
+the context of this issue.  For example feature requests may not need a
+FreeOrion version, but maybe depend on the operating system you use.  When in
+doubt provide all information you have available.
+
+You can find the FreeOrion version number in the lower right corner of the game
+main screen.
+
+When stating the operating system also note the version of the operating system
+so, `Windows 8 Pro` instead of `Windows` or `Mac OSX Mountain Lion` instead of
+`OSX`.
+-->
+
+* **FreeOrion Version**:
+* **Operating System**:
+* **Graphic card used**:
+* **Fetched as**
+  * [ ] Binary release
+  * [ ] Compiled from source
```

Comment:
This won't work because the bug reporter needs a GitHub login to report those issues. So filling out a issue report just to be put in front of a login form would be a sure way to infuriate the bug reporter as he already has invested to work out the report.

Also I think it's a bad idea because the user is not encouraged to search if his issue was already reported.